Gruntfile.js

```diff
@@ -138,7 +138,19 @@ module.exports = function (grunt) {
             src: ['omero_figure/static/figure/3rdparty/**',
                   'omero_figure/static/figure/css/**',
                   'omero_figure/static/figure/images/**',
-                  'omero_figure/static/figure/templates.js'], dest: 'demo/'},
+                  'omero_figure/static/figure/templates.js'],
+            dest: 'demo/'
+          },
+        ]
+      },
+      shapeEditor: {
+        files: [
+          {
+            expand: true,
+            cwd: 'node_modules/ome-shape-editor/dist/js',
+            src: 'shape-editor.js',
+            dest: 'omero_figure/static/figure/3rdparty/shape-editor-3.1.0'
```

Comment:
The version should ideally be retrieved from package.json

Reply:
Hmm - that might be kinda tricky.
This version number is also in index.html
```
<script src="{% static 'figure/3rdparty/shape-editor-3.1.0/shape-editor.js' %}"></script>
```
Having that version number in multiple places is not new, but now we don't have the shape-editor.js code itself.